bench/perf_run_py/perf_run.py

```diff
@@ -79,17 +75,15 @@ def get_git_info(git_working_tree_dir):
     return git_info
 
 
-def construct_command_line(exec_path: str, test: str, home: str, arguments: List[str]):
+def construct_command_line(exec_path: str, test: List[str], home: List[str], arguments: List[str]):
```

Comment:
Minor: change these to `test_arg` and `home_arg`

Reply:
Updated in [11b9a6c](https://github.com/wiredtiger/wiredtiger/pull/7223/commits/11b9a6ccbcac1c27ad8466448b690cbbce3b52d0)